pipeline/common-uncovered-coverage.py

```diff
@@ -18,26 +18,27 @@ def main():
     parser.add_argument('outfile')
 
     args = parser.parse_args()
-    fp =open(args.outfile, 'a+') 
+    fp =open(args.outfile, 'w') # @CTB changed from an 'at'
 
     genome_dict1 = {}
     genome_uncov = {}
     for record in screed.open(args.genome):
-        genome_dict1[record.name] = [0] * len(record.sequence)  
+        name = record.name.split(' ')[0]
+        genome_dict1[name] = [0] * len(record.sequence)  
     
     c = 0
     for line in open(args.uncov):
+        line = line.strip()
 	if line.startswith('>'): 
-             line.split('>') 
-             name = line[:-1]
-	     name = name[1:] 
+	     name = line[1:] 
 	else:
              indeces = line.split(',')
-             indeces = indeces[:-1]
-             indeces = [int(i) for i in indeces]
+             indeces = [int(i) for i in indeces if i]
              genome_uncov[name] = indeces 
 	     c += len(indeces) 
 
+    print 'done reading %s - %d uncov total.' % (args.uncov, c)
+
```

Comment:
We have no. of common uncovered bases reported in the output of analyze_assembly, but it much better to have it here.